README.ES.md

```diff
@@ -0,0 +1,196 @@
+# ***Plex Media Server and Plexdrive 🐳***
+
+<div align="center"><img src="https://raw.githubusercontent.com/rapejim/pms-plexdrive-docker/public/images/banner.png" width="50%"></div>
+
+Combina el poder de **Plex Media Server** *(por sus siglas en Inglés PMS)* con
+los archivos multimedia en tu cuenta de Google Drive (o
+[unidad compartida de equipo](https://support.google.com/a/users/answer/9310156?hl=es))
```

Comment:
Hey @rapejim , this was the only thing I was not 100% about. Were you refering to this when you mentioned "Team drive"? Or is it something else?

Reply:
Translate it for "Unidad Compartida", because Team Drive was the old name, the new name in English is Shared Drive.

Reply:
Done. I updated the "Team Drive" references on the main `README` as well.